openapi/mt_openapi_spec_v1.yaml

```diff
@@ -12077,6 +12081,21 @@ components:
           description: An array of invoice line items. The API supports a maximum
             of 50 invoice line items per invoice. If a greater number of invoice line
             items is required, please contact support.
+        auto_advance:
+          type: boolean
+          nullable: true
+          description: 'Translation missing: en.openapi.descriptions.invoice.schema.auto_advance'
```

Comment:
Missing translation, do not merge

Reply:
I'm noticing we actually have a number of missing translations for legal entity risk rating from 4 mo ago too

Reply:
https://github.com/Modern-Treasury/monorepo/pull/27364